test/distributed/org/apache/cassandra/distributed/test/CASTest.java

```diff
@@ -79,8 +79,9 @@ public static void beforeClass() throws Throwable
                                                    .set("write_request_timeout", REQUEST_TIMEOUT)
                                                    .set("cas_contention_timeout", CONTENTION_TIMEOUT)
                                                    .set("request_timeout", REQUEST_TIMEOUT);
-        THREE_NODES = init(Cluster.create(3, conf));
-        FOUR_NODES = init(Cluster.create(4, conf), 3);
+        // TODO: fails with vnode enabled
```

Comment:
Should we file a jira for this? I can't see anything in this test that should make it fail with vnodes, could be a bug or test bug?